client/src/panels/panel_declarations/covid/covid_expenditures.js

```diff
@@ -399,49 +343,49 @@ class CovidExpendituresPanel extends React.Component {
     super(props);
     this.state = {
       loading: true,
+      summary_by_fiscal_year: null,
+      selected_year: _.last(props.panel_args.years),
     };
   }
   componentDidMount() {
-    client
-      .query({
-        query: gov_covid_summary_query,
-        variables: {
-          lang: lang,
-          _query_name: "gov_covid_summary_query",
-        },
+    query_gov_covid_summaries().then((covid_summaries) =>
```

Comment:
I'm only checking how queries are being done in this PR right now, but I think using hooks is the superior way if querying data panel by panel basis for a few reasons. Mainly, hooks take care of `loading`, as well as state management (since there is no need to do any mostly) and also less boilerplate code. Just my 2 cents after trying out hooks, and taking a super quick look on how queries are being done here